tests/unit/utils/test_qtutils.py

```diff
@@ -108,9 +108,9 @@ def parser(self, mocker):
         # No Qt arguments
         (['--debug'], [sys.argv[0]]),
         # Qt flag
-        (['--debug', '--qt-reverse', '--nocolor'], [sys.argv[0], '-reverse']),
+        (['--debug', '--qt-flag', '--nocolor', 'reverse'], [sys.argv[0], '-reverse']),
```

Comment:
The `reverse` should be directly after the `--qt-flag` here, without any other flag in between.

This made this test fail, but for some reason the relevant output is between *a lot* of other stuff: https://travis-ci.org/The-Compiler/qutebrowser/jobs/158660931#L1423